sig/ffi/abstract_memory.rbs

```diff
@@ -159,6 +159,6 @@ module FFI
     def write_array_of_long_long: (Array[Integer] ary) -> self
     def write_array_of_float: (Array[Numeric] ary) -> self
     def write_array_of_double: (Array[Numeric] ary) -> self
-    def write_array_of_pointer: (Array[Pointer::_ToPtr] ary) -> self
+    def write_array_of_pointer: (Array[Pointer::_ToPtr?] ary) -> self
```

Comment:
Can you please decide on your own?

Reply:
It’s implicit, but `#(put|write)_array_of_pointer` also accepts Integer elements (*but not `#to_int`s* since they share the conversion implementation with `#(put|write)_pointer`.

---

While [FFI currently only mention `#to_ptr` in `AbstractMemory`](https://github.com/search?q=repo%3Affi%2Fffi%20%22%23to_ptr%22&type=code), the `_ToPtr|Integer` pattern is useful in user libraries’ (i.e., bindings based on FFI) RBS as well, so I’ll surfacing the type alias to the `FFI` namespace.